src/js/user-profile/reducers/login.js

```diff
@@ -0,0 +1,33 @@
+import _ from 'lodash';
+import lodashDeep from 'lodash-deep';
+
+import { UPDATE_LOGGEDIN_STATUS, UPDATE_LOGIN_URL } from '../actions';
+
+// Add deep object manipulation routines to lodash.
+_.mixin(lodashDeep);
+
+
+const initialState = {
+  currentlyLoggedIn: false,
+  loginUrl: '',
+};
+
+function loginStuff(state = initialState, action) {
+  let newState = undefined;
+  switch (action.type) {
+    case UPDATE_LOGGEDIN_STATUS:
+      newState = Object.assign({}, state);
+      _.set(newState, 'currentlyLoggedIn', action.value);
```

Comment:
@crwallace  What are your thoughts behind using lodash here instead of vanilla javascript methods?
e.g.
```javascript
return {
  ...state,
  currentlyLoggedIn: action.value,
}
```